### ISSUE TITLE
When trying to export or import, regardless of DB type, Mileage crashes

### ISSUE THREAD

**Issue by GoogleCodeExporter** [evancharlton/android-mileage#1]
```
What steps will reproduce the problem?
1. Open Mileage, hit Menu and select Export/Import
2. Choose any of the options there, SQLite/SQL/CSV on either Export or Import 
and it crashes

What is the expected output? What do you see instead?
It should prompt for the import or export file to be saved or loaded. Instead, 
I get Force Close.

What version of the product are you using? On what operating system?
Version 2.2.5

HTC Evo, Android 2.2

Please provide any additional information below.


```

Original issue reported on code.google.com by `Jmichels...@gmail.com` on 25 Oct 2010 at 9:04

**Comment by GoogleCodeExporter** [comment]
```
Just to clarify, this happens before you get a screen like the attached 
screenshot?
```

Original comment by `evancharlton` on 25 Oct 2010 at 10:51

Attachments:
- [export.png](https://storage.googleapis.com/google-code-attachments/android-mileage/issue-1/comment-1/export.png)

**Comment by GoogleCodeExporter** [comment]
```
Yep, it is before that screen. The second I hit any of the 6 options in that 
window, it crashes.

I'm not sure what caused this issue, but rebooting my phone fixed it for the 
time being. I was able to successfully export.
```

Original comment by `Jmichels...@gmail.com` on 25 Oct 2010 at 10:56

**Comment by GoogleCodeExporter** [comment]
```
Was your phone plugged in to your computer at the time or anything? I think I 
know where the problem is in the code, but I'd like to track down the root 
cause.
```

Original comment by `evancharlton` on 26 Oct 2010 at 12:00

**Comment by GoogleCodeExporter** [comment]
```
The phone was unplugged from the computer.
```

Original comment by `Jmichels...@gmail.com` on 26 Oct 2010 at 1:46

**Comment by GoogleCodeExporter** [comment]
```
Strange. Well, I'll get a fix uploaded tonight if possible. Definitely within 
the next 24 hours.
```

Original comment by `evancharlton` on 26 Oct 2010 at 2:18
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Thank you for the attentive support. If I am the only one reporting the issue, 
there is no rush. Like I said, rebooting the phone solved the problem for the 
time being.
```

Original comment by `Jmichels...@gmail.com` on 26 Oct 2010 at 2:31

**Comment by GoogleCodeExporter** [comment]
```
You're not the only one, you're just the first one who caught me when I 
happened to have some free time :-)
```

Original comment by `evancharlton` on 26 Oct 2010 at 2:32

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 2.2.6. Please let me know if you see this again.
```

Original comment by `evancharlton` on 27 Oct 2010 at 3:24
- Changed state: **Fixed**
